nopremium.js

```diff
@@ -1,21 +1,51 @@
-const elements = [
-  { selector: 'a[href="https://users.nexusmods.com/account/billing/premium"]', removeParent: true },
-  { selector: '.premium-block', removeParent: false },
-  { selector: '.premium-banner', removeParent: false },
-  { selector: '#rj-vortex', removeParent: false },
-  { selector: '.subheader, table thead, table tbody, #fastDownloadButton', removeParent: false }
+const elements = [{
+        selector: 'a[href="https://users.nexusmods.com/account/billing/premium"]',
+        removeParent: true
+    },
+    {
+        selector: '.premium-block',
+        removeParent: false
+    },
+    {
+        selector: '.premium-banner',
+        removeParent: false
+    },
+    {
+        selector: '#rj-vortex',
+        removeParent: false
+    },
+    {
+        selector: 'table', // Only select tables and handle the rest in the script
+        removeParent: false
+    }
 ];
 
 function removeElements() {
-  elements.forEach((el) => {
-    document.querySelectorAll(el.selector).forEach((matched) => {
-      if (el.removeParent) {
+    elements.forEach((el) => {
+        if (el.selector === 'table') {
+            document.querySelectorAll(el.selector).forEach((table) => {
+                const thContainsText = Array.from(table.querySelectorAll('thead th')).some(th => th.textContent.trim() === 'Choose download type');
```

Comment:
Which element was this conflicting with? I don't like string matching like this since it's dependent on the user's language and can be quite fragile.

Reply:
It was the last one. It selects all tables, but Vortex uses tables in the examples mentioned in #1. I wasn't able to get it to work with just a selector, but hopefully you might be able to :).

Reply:
Another note: this approach sadly doesn't remove the `Choose from the options below` text. There might be a solution for that as well, but I haven't found it yet. Nexusmods basically has no proper DOM labeling, so I'm unsure if there's a selector that'll work flawlessly.